### PR TITLE
Fix indents (and two more `_decode`s)

### PR DESCRIPTION
**Pull request by jean**

### CODE REVIEW

File: git-hub
Comment (daniel-zullo) [inline git-hub]: I think the change related to `urllib` should go in the commit `Remove Python 3-fix that snuck in` as it isn't what the commit `POST data should be bytes` tries to address

File: git-hub
Comment (daniel-zullo) [inline git-hub:1720]: I am not sure if it is suggested to `()` in this case. Just checking...

File: git-hub
Comment (daniel-zullo) [inline git-hub]: I wonder if this line can join the line above (concatenate them) as it might look better

**Review by daniel-zullo — COMMENTED** [review]
I only have a comment about `urllib` and LGTM

**Comment by daniel-zullo** [comment]
There is also a problem with python 2, you can reproduce it trying to create a new PR

e.g. `git hub pull new -b master -c fix-version`

```
Traceback (most recent call last):
  File "/usr/bin/git-hub", line 2192, in <module>
    main()
  File "/usr/bin/git-hub", line 2184, in main
    args.run(parser, args)
  File "/usr/bin/git-hub", line 670, in check_config_and_run
    cmd.run(parser, args)
  File "/usr/bin/git-hub", line 1515, in run
    args.assignee, args.milestone)
  File "/usr/bin/git-hub", line 1302, in _do_update
    issue = req.patch(url, **params)
  File "/usr/bin/git-hub", line 507, in <lambda>
    self.json_req(url, method, *args, **kwargs)
  File "/usr/bin/git-hub", line 483, in json_req
    res = self.auth_urlopen(url, method, body)
  File "/usr/bin/git-hub", line 427, in auth_urlopen
    req = urllib2.Request(url, _encode(data))
  File "/usr/bin/git-hub", line 68, in _encode
    return _string.encode(encoding, 'replace')
AttributeError: 'NoneType' object has no attribute 'encode'
```

File: git-hub
Comment (jean) [reply git-hub]: Yeah sorry, I removed that in https://github.com/sociomantic-tsunami/git-hub/pull/272/commits/0720d88b681cde602b5f4dffa5c399d4ffad9935

File: git-hub
Comment (jean) [reply git-hub:1720]: That's really just a personal preference .. I prefer using `(...)` to split an expression across lines, rather than a backslash.

File: git-hub
Comment (jean) [reply git-hub]: Agreed, https://github.com/sociomantic-tsunami/git-hub/pull/272/commits/3c099c443067b74da474efa09c8e9ecd54a9d2f1

**Comment by jean** [comment]
@daniel-zullo-sociomantic could you maybe have another look? I think it should be OK now ..

**Review by daniel-zullo — APPROVED** [review]
LGTM.

I've tested this patch with a couple pull commands and works fine. I'll ask to someone with right permissions to review and merge it. After this patch gets merged we can create a release candidate (pre-release) and tested it again.

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**
